wizrb.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split('\x0').reject do |f|
-      (f == __FILE__) || f.match(%r{\A(?:(?:test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{\A(?:(?:test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
     end
```

Comment:
It might be worth while just being explicit instead of doing all of this default mumbo jumbo.
```ruby
spec.files = Dir["CHANGELOG.md", "LICENSE.txt", "README.md", "doc/*", "exe/*", "lib/**/*"]
```

Reply:
I agreed. I've committed the changes.